samples/military_feature_schemas/comparison_test_results/tested_Fields_ControlMeasuresAreas.csv

```diff
@@ -0,0 +1,26 @@
+field_name,field_type,field_length,field_alias,nullability,field_domain,field_default,field_notes,field_comparison_result
+identity,Integer,4,Standard Identity,NULLABLE,Identity,,,OK
+symbolset,SmallInteger,2,Symbol Set,NULLABLE,ControlMeasuresAreas,25,,ControlMeasuresAreas ; 25
+entity,Integer,4,Control Measures Area,NULLABLE,Control_Measure_Area_Entities,,,OK
+echelon,Integer,4,Echelon,NULLABLE,Echelons,,,Echelon
+uniquedesignation,String,30,Unique Designation,NULLABLE,,,,50
+additionalinformation,String,50,Additional Information,NULLABLE,,,,OK
+additionalinformation2,String,50,Additional Information 2,NULLABLE,,,,OK
+distance,Double,8,Distance (meters),NULLABLE,,,,hostile ; String ; 3 ; Hostile (ENY)
+azimuth,SmallInteger,2,Azimuth (degrees),NULLABLE,,,,type ; String ; 50 ; Type of Equipment
+distance2,Double,8,Distance 2 (meters),NULLABLE,,,,distance ; Distance (meters)
+targetdesignator,String,6,Target Designator,NULLABLE,,,,distance2 ; Double ; 8 ; Distance 2 (meters)
+hostile,String,3,Hostile (ENY),NULLABLE,,,,azimuth ; SmallInteger ; 2 ; Azimuth (degrees)
+type,String,24,Type of Equipment,NULLABLE,,,,targetdesignator ; 50 ; Target Designator
+x,Double,8,X Location,NULLABLE,,,,OK
+y,Double,8,Y Location,NULLABLE,,,,OK
+z,Double,8,Altitude/Depth,NULLABLE,,,,OK
+x2,Double,8,X Location 2,NULLABLE,,,,z2 ; Altitude/Depth 2
```

Comment:
The addition of lines in the new proposed schema files, or a change in line order, results in differences being encountered when doing a token comparison between two separate comma delimited tokenized lines.  The intent of this test was to provide some automated way of highlighting those differences, so the eye could spot them without having to look at each pair of lines in two separate files.  .  .